docs/update-from-client-side-helpers-to-server-side.md

```diff
@@ -0,0 +1,229 @@
+# Upgrading your Client-side integration to a Server-side integration
+
+We recommend using a combination of client-side and server-side code to integrate the PayPal Buttons on your e-commerce website. If you currently have client-side only PayPal Buttons integration, this page will describe how to update your integration to the recommended approach.
+<br/><br/>
+> 
+> **Know before you code**
```

Comment:
```suggestion
## Know before you code**
```

Don't have to take this suggestion but I was wondering if this looked better as a heading. Any thoughts? I'm on the fence between the two but slightly leaning towards heading

Reply:
I think we want this section to be less important than the "What do I need to change?" section. An h2 could work or we bold it. I'm flexible here as well.

Reply:
Ah yeah that makes sense. I honestly don't have a strong opinion just an idea. I vote lets keep it as is then. So I'll resolve this comment as won't fix haha.